### DISCUSSION TITLE
debezium CDC to Redis

### DISCUSSION THREAD

**Discussion by dgreco3917** [jaredpetersen/kafka-connect-redis#33] [Q&A]
Looking for advice on how to enable CDC events from Debezium to sink to Redis. These events are being sourced from MySQL. How should I be converting from CDC events to Redis commands? 

By comparison, I am currently sinking these events to Elastic using the following config:

            "transforms": "unwrap,key",
            "transforms.unwrap.type": "io.debezium.transforms.ExtractNewRecordState",
            "transforms.key.type": "org.apache.kafka.connect.transforms.ExtractField$Key",
            "transforms.key.field": "SubscriberID",
            "key.ignore": "false",
            "type.name": "subscribers"

**Comment by jaredpetersen** [comment]
Your best option is probably to build a [Kafka Streams](https://kafka.apache.org/documentation/streams/) application as a middleman. This would allow you to perform more complex transformations than what can be accomplished with Kafka Connect Single Message Transforms (SMTs).
